index.html

```diff
@@ -1,13 +1,18 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
-</head>
-<body>
-  <h1>Best Coffee Ever!</h1>
-  <h3>Heres a list of some of the best coffee ever</h3>
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <h1>Best Coffee Ever!</h1>
+    <h3>Heres a list of some of the best coffee ever</h3>
+    <ul>
+      <li>Lava Java - Peaberry</li>
+      <li>8th & Roash - Jetsetter</li>
```

Comment:
Please fix this spelling error

Reply:
Please review changes once more, I fixed it.